### ISSUE TITLE
Errors occur in the init command when member account names are duplicated?

### ISSUE THREAD

**Issue by NobuhiroNakayama** [OlafConijn/AwsOrganizationFormation#41]
An error occurred in the init command when member accounts with the same name was included in the organization.

The status of the organization when the error occurred is as follows.
The member accounts have same name.

```
$ aws organizations list-accounts --profile organizations
{
    "Accounts": [
        {
            "Status": "ACTIVE",
            "Name": "Master",
            "Email": "********+master@gmail.com",
            "JoinedMethod": "INVITED",
            "JoinedTimestamp": 1573634525.57,
            "Id": "XXXXXXXXXXXX",
            "Arn": "arn:aws:organizations::XXXXXXXXXXXX:account/o-xxxxxxxxxx/XXXXXXXXXXXX"
        },
        {
            "Status": "ACTIVE",
            "Name": "Nobuhiro Nakayama",
            "Email": "********+001@gmail.com",
            "JoinedMethod": "CREATED",
            "JoinedTimestamp": 1573634631.151,
            "Id": "YYYYYYYYYYYY",
            "Arn": "arn:aws:organizations::XXXXXXXXXXXX:account/o-xxxxxxxxxx/YYYYYYYYYYYY"
        },
        {
            "Status": "ACTIVE",
            "Name": "Nobuhiro Nakayama",
            "Email": "********+002@gmail.com",
            "JoinedMethod": "CREATED",
            "JoinedTimestamp": 1573634646.563,
            "Id": "ZZZZZZZZZZZZ",
            "Arn": "arn:aws:organizations::XXXXXXXXXXXX:account/o-xxxxxxxxxx/ZZZZZZZZZZZZ"
        }
    ]
}
```

The error message is as follows.

```
$ org-formation init organization.yml --region us-east-1 --profile organizations --print-stack
ERROR: unexpected error occurred...
duplicated mapping key at line 22, column -193:
      NobuhiroNakayamaAccount:
      ^
YAMLException: duplicated mapping key at line 22, column -193:
      NobuhiroNakayamaAccount:
      ^
    at generateError (/usr/local/lib/node_modules/aws-organization-formation/node_modules/js-yaml/lib/js-yaml/loader.js:167:10)
    at throwError (/usr/local/lib/node_modules/aws-organization-formation/node_modules/js-yaml/lib/js-yaml/loader.js:173:9)
    at storeMappingPair (/usr/local/lib/node_modules/aws-organization-formation/node_modules/js-yaml/lib/js-yaml/loader.js:335:7)
    at readBlockMapping (/usr/local/lib/node_modules/aws-organization-formation/node_modules/js-yaml/lib/js-yaml/loader.js:1098:9)
    at composeNode (/usr/local/lib/node_modules/aws-organization-formation/node_modules/js-yaml/lib/js-yaml/loader.js:1359:12)
    at readBlockMapping (/usr/local/lib/node_modules/aws-organization-formation/node_modules/js-yaml/lib/js-yaml/loader.js:1089:11)
    at composeNode (/usr/local/lib/node_modules/aws-organization-formation/node_modules/js-yaml/lib/js-yaml/loader.js:1359:12)
    at readDocument (/usr/local/lib/node_modules/aws-organization-formation/node_modules/js-yaml/lib/js-yaml/loader.js:1519:3)
    at loadDocuments (/usr/local/lib/node_modules/aws-organization-formation/node_modules/js-yaml/lib/js-yaml/loader.js:1575:5)
    at load (/usr/local/lib/node_modules/aws-organization-formation/node_modules/js-yaml/lib/js-yaml/loader.js:1596:19)
```

I was able to avoid the error by eliminating duplicate account names.
(I canceled one of member accounts)

Is this a specification or constraint? Is it a bug?

**Comment by OlafConijn** [comment]
thanks! that would be a bug, will soon be fixed

**Comment by OlafConijn** [comment]
fixed this in the commit above. will let you know when released.
generating the template will not fail, it will print a warning though:
`WARN: Multiple accounts found with AccountName YourNameHere. This will not be a problem, but perhaps confusing?`

does that make sense?

**Comment by NobuhiroNakayama** [comment]
I think it is a good error message.

Account names within an organization are not required to be unique.
https://docs.aws.amazon.com/organizations/latest/APIReference/API_CreateAccount.html
However, if there are many accounts with the same name, it is difficult to identify the account on the management console.

**Comment by OlafConijn** [comment]
closed in version 0.9.3
